client/my-sites/sidebar/index.jsx

```diff
@@ -1149,12 +1152,8 @@ function mapStateToProps( state ) {
 		isAtomicSite: !! isSiteAutomatedTransfer( state, selectedSiteId ),
 		isMigrationInProgress,
 		isVip: isVipSite( state, selectedSiteId ),
-		showCustomizerLink:
-			! (
-				isSiteUsingFullSiteEditing( state, selectedSiteId ) ||
-				isSiteUsingCoreSiteEditor( state, selectedSiteId )
-			) && siteId,
-		showSiteEditor: isSiteUsingCoreSiteEditor( state, selectedSiteId ),
+		showCustomizerLink: ! isSiteUsingFullSiteEditing( state, selectedSiteId ) && siteId,
```

Comment:
Note that the logic is updated here to show the Customizer link in the sidebar for sites using Core FSE. All the block-based themes we have on Simple sites right now are universal themes, which expect both the Customizer and the Site editor to be available.